external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -1497,6 +1497,32 @@ abstract class AvroSuite extends QueryTest with SharedSparkSession {
       |}
     """.stripMargin)
   }
+
+  test("log a warning of ignoreExtension deprecation") {
+    val logAppender = new LogAppender
+    withTempPath { dir =>
+      Seq(("a", 1, 2), ("b", 1, 2), ("c", 2, 1), ("d", 2, 1))
+        .toDF("value", "p1", "p2")
+        .repartition(2)
+        .write
+        .format("avro")
+        .option("header", true)
+        .save(dir.getCanonicalPath)
+      withLogAppender(logAppender) {
+        spark
+          .read
+          .format("avro")
+          .option(AvroOptions.ignoreExtensionKey, false)
+          .option("header", true)
```

Comment:
Does Avro has header option?

Reply:
I copy pasted the piece of code from another test in AvroSuite: https://github.com/apache/spark/blob/94fc0e3235162afc6038019eed6ec546e3d1983e/external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala#L1524 . Let me check and remove it if is not needed.

Reply:
Yeh, it is not needed. It seems the test which I copy-pasted was copy-pasted from another places. I guess from https://github.com/apache/spark/blob/b389b8c5f0650a7e63098f18437fcaa29998732a/sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala#L746-L787

Let me remove the option from other avro test as well.